src/coreclr/vm/methodtable.cpp

```diff
@@ -8042,21 +8057,19 @@ MethodTable::MethodData *MethodTable::GetMethodDataHelper(MethodTable *pMTDecl,
 }
 
 //==========================================================================================
-// The fCanCache argument determines if the resulting MethodData object can
+// The computeOptions argument determines if the resulting MethodData object can
 // be added to the global MethodDataCache. This is used when requesting a
 // MethodData object for a type currently being built.
 MethodTable::MethodData *MethodTable::GetMethodData(MethodTable *pMTDecl,
                                                     MethodTable *pMTImpl,
-                                                    BOOL fCanCache)
+                                                    MethodDataComputeOptions computeOptions)
 {
     CONTRACTL {
         THROWS;
         WRAPPER(GC_TRIGGERS);
     } CONTRACTL_END;
```

Comment:
```suggestion
    WRAPPER_NO_CONTRACT;
```

This can probably be updated to a wrapper contract.